brainglobe_atlasapi/atlas_generation/atlas_scripts/praire_vole.py

```diff
@@ -0,0 +1,256 @@
+__version__ = "0"
+
+import os
+import random
+import shutil
+import time
+import zipfile
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pooch
+import tifffile
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+def create_atlas(working_dir):
+    ATLAS_NAME = "..."
+    SPECIES = "..."
+    ATLAS_LINK = "https://elifesciences.org/articles/87029#data"
+    CITATION = "..."
+    ATLAS_BASE_URL = "https://ndownloader.figshare.com/files/"
+    ORIENTATION = "..."
```

Comment:
This is the orientation of the raw data (before packaging), and should be `asr`. Details [here](https://brainglobe.info/documentation/setting-up/image-definition.html) if you're interested.